### PR TITLE
feat: Update READMEs with Secret Manager info

### PR DESCRIPTION
**Pull request by averikitsch**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: cloud-sql/mysql/sqlalchemy/README.md [hunk 1]
```diff
@@ -177,6 +177,26 @@ instance configuration.
 
 This step can be done as part of deployment but is separated for clarity.
 
+It is recommended to use the [Secret Manager integration](https://cloud.google.com/run/docs/configuring/secrets) for Cloud Run instead
+of using environment variables for the SQL configuration. The service injects the SQL credentials from
+Secret Manager at runtime via an environment variable.
+
+Create secrets via the command line:
+```sh
+echo -n "my-awesome-project:us-central1:my-cloud-sql-instance" | \
```
Comment (dandhlee) [inline cloud-sql/mysql/sqlalchemy/README.md:186]: does the user's project name and sql instance have to be inputted into the `my-awesome-project:us-central1:my-cloud-sql-instance` echo? If so, can we make it consistent with the other indications like `[YOUR_PROJECT_ID]`?

File: cloud-sql/postgres/sqlalchemy/README.md [hunk 2]
```diff
@@ -175,6 +175,26 @@ instance configuration.
 
 This step can be done as part of deployment but is separated for clarity.
 
+It is recommended to use the [Secret Manager integration](https://cloud.google.com/run/docs/configuring/secrets) for Cloud Run instead
+of using environment variables for the SQL configuration. The service injects the SQL credentials from
+Secret Manager at runtime via an environment variable.
+
+Create secrets via the command line:
+```sh
+echo -n "my-awesome-project:us-central1:my-cloud-sql-instance" | \
```
Comment (dandhlee) [inline cloud-sql/postgres/sqlalchemy/README.md:184]: ditto

**Review by dandhlee — CHANGES_REQUESTED** [review]
Looks good, minor comment. Please see below!

File: cloud-sql/mysql/sqlalchemy/README.md (hunk 1 above)
Comment (averikitsch) [reply cloud-sql/mysql/sqlalchemy/README.md]: Used the env var above

**Review by dandhlee — APPROVED** [review]
LGTM!

**Review by parthea — APPROVED** [review]

**Pull request merged**